js/src/APIClient.js

```diff
@@ -165,6 +165,26 @@ export class APIClient {
       }
     );
   }
+  async prepareSignature(contractId:string, fullName:string, accessToken:?string):Promise<object> {
+      return this.getCall('/signForMetamask/contract/' + contractId,  {
+          fullName,
+          accessToken
+      });
+  }
+
+  async signContract(contractId:string, fullName:string, accessToken:?string):Promise<object> {
+      return this.getCall('/sign/contract/' + contractId, {
+          fullName,
+          accessToken
+      });
+  }
+
+  async loadContractStatus(contractId:string, accessToken:?string):Promise<object> {
+      return this.getCall('/contract/sign/status', {
+          id: contractId,
+          accessToken
+      }).then(result => result.data);
```

Comment:
in routes in your related pr in openlaw backend https://github.com/openlawteam/openlaw/pull/1689/files i noticed that this GET request has only the `id` parameter. should that be updated to include the accessToken too?

Reply:
you're right! fixing it on openlaw